packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -41,580 +41,190 @@ export const Default = {
-          width: 100%;
-          margin-bottom: 10px;
-          box-sizing: border-box;
-        }
+      <table class="sd-table">
+        <tbody>
+          <tr class="relative">
+            <td class="sd-table-cell sd-table-cell--bg-transparent">bg-transparent</td>
+            <td class="sd-table-cell sd-table-cell--bg-white">bg-white</td>
+            <td class="sd-table-cell sd-table-cell--bg-primary-100">bg-primary-100</td>
+            <td class="sd-table-cell sd-table-cell--bg-neutral-100">bg-neutral-100</td>
+          </tr>
+        </tbody>
+      </table>
+    `;
+  }
+};
 
-        .sample-table {
-          margin-top: 20px;
-          width: 100%;
-        }
+/**
+ * Use the `&--divider` class to toggle vertical dividers.
+ */
 
-        .sd-table-cell.sortable {
-          padding: 0;
-        }
+export const Divider = {
+  render: () => {
+    return html`
+      <table class="sd-table">
+        <thead>
+          <tr>
+            <td class="font-bold sd-table-cell sd-table-cell--bg-transparent sd-table-cell--divider">Header</td>
+          </tr>
+        </thead>
+        <tr class="relative">
+          <td class="sd-table-cell sd-table-cell--bg-transparent sd-table-cell--divider">Cell content</td>
+        </tr>
+
+        <tr class="relative">
+          <td class="sd-table-cell sd-table-cell--bg-transparent sd-table-cell--divider">Cell content</td>
+        </tr>
+      </table>
+    `;
+  }
+};
 
-        .sd-table-cell.sortable button {
-          display: inline-flex;
-          box-sizing: border-box;
-          height: 100%;
-          width: 100%;
-          margin: 0;
-          padding: 16px;
-          align-items: center;
-          justify-content: flex-start;
-          gap: 8px;
-          text-align: left;
-        }
+/**
+ * Use `&--shadow-*` to add a shadow to columns or rows. This is especially useful when the table is scrollable with sticky headers.
+ */
 
-        div:not(:first-of-type).headline {
-          margin-top: 72px;
-        }
-      </style>
-      <div class="story-wrapper">
-        <div class="headline">Simple Table</div>
-        <table class="sd-table sample-table">
-          <thead>
-            ${(() => {
-              return html`<tr>
-                ${headerData.map(cellData => {
-                  return html`<th class="sd-table-cell sd-table-cell--bg-white">${cellData}</th>`;
-                })}
-              </tr>`;
-            })()}
-          </thead>
-          <tbody>
-            ${tableData.map(rowData => {
-              return html`<tr>
-                ${rowData.map(cellData => {
-                  return html`<td class="sd-table-cell sd-table-cell--bg-transparent">${cellData}</td>`;
-                })}
-              </tr>`;
-            })}
-          </tbody>
-        </table>
-        <div class="headline">Simple Table With Vertical Lines</div>
+export const Shadow = {
+  render: () => {
+    return html`
+      <div class="flex gap-16 flex-wrap">
```

Comment:
Could we show it like here?

![image](https://github.com/user-attachments/assets/df93b582-b9cf-4056-9678-19fe022d6b52)